backstop/tests/breadcrumbs.html

```diff
@@ -84,7 +84,7 @@ <h1>Breadcrumbs</h1>
               <svg-chevron-right aria-hidden="true"></svg-chevron-right>
             </li>
             <li class="iui-breadcrumbs-item">
-              <span class="iui-ellipsis">&mldr;</span>
+              <span>&mldr;</span>
```

Comment:
This class was not doing anything?

Reply:
There are no rules for it, so I guess not /